src/test/java/net/revelc/code/formatter/xml/lib/FormatterTest.java

```diff
@@ -6,20 +6,37 @@
  * which is available at https://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *     Saikiran Revuru - Added test case testDeleteBlankLines 
  *******************************************************************************/
 package net.revelc.code.formatter.xml.lib;
 
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
 import org.junit.jupiter.api.Test;
 
 public class FormatterTest {
+    
+    @Test
+    public void testDeleteBlankLines() throws IOException {
+	FormattingPreferences prefs = new FormattingPreferences();
+	prefs.setDeleteBlankLines(true);
+	XmlDocumentFormatter formatter = new XmlDocumentFormatter(System.lineSeparator(), prefs);
+
+	String inXml = new String(Files.readAllBytes(Paths.get("src/test/resources/test-space-input.xml")),
+		StandardCharsets.UTF_8);
```

Comment:
Good test case. I have two suggestions:

1. replace tabs with spaces (the formatter should do this when the project is built; having the indentation/formatting correct in the PR avoids a second follow-up commit to clean up the code style)
2. a static import for `UTF_8` can make things easier to read; there are few things I would recommend a static import for, but the StandardCharsets fields are definitely on my list (JUnit assert methods being another)